modules/claims_api/lib/claims_api/error/soap_error_handler.rb

```diff
@@ -8,44 +8,39 @@ class SoapErrorHandler
     # list faraday errors: https://lostisland.github.io/faraday/middleware/raise-error
 
     def handle_errors(response)
-      hash = Hash.from_xml(response.body)
+      @hash = Hash.from_xml(response.body)
 
-      return if hash&.dig('Envelope', 'Body', 'Fault').blank?
+      return if @hash&.dig('Envelope', 'Body', 'Fault').blank?
 
-      fault_description = get_fault_description(hash)
+      # catch message and log it with claim id
+      fault_description = get_fault_description
 
       get_exception(fault_description)
     end
 
-    def get_fault(hash)
-      fault_code = hash&.dig('Envelope', 'Body', 'Fault', 'faultcode')
+    def get_fault
+      fault_code = @hash&.dig('Envelope', 'Body', 'Fault', 'faultcode')
       fault_code.split(':')[1]
```

Comment:
Not a part of this code change, but if the `.dig` returns nil, then `.split` (and `[1`]) will fail - I'm guessing `fault_code&.split(':')&.dig(1)` may work?